modules/javafx.web/src/test/java/test/javafx/scene/web/MathMLRenderTest

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
This test file's extension is missing. Did you run this?

It should be MathMLRenderTest.java right?

Reply:
Yes, the name is MathMLRenderTest.java
I ran the test but with my working copy and pasted the code directly online in the GitHub editor.
I forgot to add the extension when I enter the name.
Sorry, I have to change my way of working.